db/schema.rb

```diff
@@ -16,7 +16,7 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-  create_table "oauth_access_grants", force: true do |t|
+  create_table "oauth_access_grants", force: :cascade do |t|
```

Comment:
I believe that changed when I pulled the project down and had to initially migrate the db. I'm not sure why that happened, though.